polkadot/node/core/candidate-validation/src/tests.rs

```diff
@@ -471,126 +472,12 @@ impl ValidationBackend for MockValidateCandidateBackend {
 	}
 }
 
-#[test]
-fn session_index_checked_only_in_backing() {
```

Comment:
removed this test because the same scenario was exercised in `invalid_session_or_core_index`